Libplanet/Blockchain/BlockChain.cs

```diff
@@ -236,8 +243,9 @@ public void Append(Block<T> block, DateTimeOffset currentTime)
                 block.Validate(
                     currentTime,
                     a => GetStates(new[] { a }, tip).GetValueOrDefault(a));
-                Validate(Enumerable.Append(this, block), currentTime);
+                Enumerable.Append(this, block);
```

Comment:
Although it's late, this seems a miss; [`Enumerable.Append()`][1] method returns a new `IEnumerable<T>` that has one more item, but this code doesn't use its return value.  According to the [docs][2]:

> This method does not modify the elements of the collection. Instead, it creates a copy of the collection with the new element.

To me, the code before changed seems correct.  @earlbread Could you fix this in the pull request #210?

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.append
[2]: https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.append#remarks

Reply:
It seems this code was changed because the type of `Validate()` method's first parameter was changed from `IEnumerable<Bock<T>>` to `IReadOnlyList<Block<T>>` in this pull request.  To fix this quickly, it could be like `Validate(Enumerable.Append(this, block).ToList(), currentTime);`, I'm skeptic if it's good though.

Reply:
This seems not needed anymore. I'll remove it in #210.